.github/workflows/aarch64.yml

```diff
@@ -0,0 +1,47 @@
+name: Ubuntu aarch64 (GCC 11)
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+
+permissions:
+  contents: read
```

Comment:
So you specified that files could not be written, correct?

Reply:
I would just drop this permission step and see whether the permission issue resolved itself.

I am not quite sure why we specify permissions... :-/

Reply:
Yes, actually I was more trying to follow the pattern I saw in other files. Also, the workflow file you shared with me as an example, it actually does not contain any permissions. I was wondering if adding permissions to write could my an issue.